packages/storefront/src/components/Playground.vue

```diff
@@ -37,7 +37,7 @@
           :frameworks="frameworks"
         ></CodeBlock>
         <CodeEditor
-          v-if="showCodeEditor"
+          v-if="showCodeEditor && activeFramework !== 'vue'"
```

Comment:
Perhaps we should disable the button instead of not rendering it anymore similar to when selecting the `Shared` tab in order to
- prevent content jumps on a page 
- give the user feedback that this isn't just missing, but instead not available, yet
<img width="374" alt="Bildschirm­foto 2023-04-06 um 10 19 10" src="https://user-images.githubusercontent.com/4380295/230318263-b2885c36-be6a-4899-b0ac-2ab486029182.png">

Reply:
Fixed it